### PR TITLE
Remove peer deps next

### PR DESCRIPTION
**Pull request by awharn**
Remove @zowe/cli peer dependency to better support NPM v7

### CODE REVIEW

**Review by gejohnston — APPROVED** [review]
Looks good.

File: package.json [hunk 1]
```diff
@@ -37,8 +37,7 @@
     "ibm_db": "2.7.1"
   },
   "peerDependencies": {
-    "@zowe/imperative": "^4.0.0",
-    "@zowe/cli": "^6.0.0"
+    "@zowe/imperative": "^4.0.0"
```
Comment (zFernand0) [inline package.json:40]: Should this versions be have a prerelease string (since this is going inot Next)?

**Review by zFernand0 — COMMENTED** [review]
Left one comment/question.

**Review by t1m0thyj — APPROVED** [review]
LGTM!

**Review by zFernand0 — APPROVED** [review]

**Pull request merged**